contracts/CurrencyNetwork.sol

```diff
@@ -348,6 +353,24 @@ contract CurrencyNetwork is CurrencyNetworkInterface, Authorizable, debtTracking
         );
     }
 
+    /**
+     * @notice `msg.sender` cancels a trustline update it initiated with _debtor
+     * @param _counterparty The other party of the trustline agreement
+     */
+    function cancelTrustlineUpdate(
```

Comment:
Maybe for consistency `cancelUpdateTrustline`?

Reply:
I would rather not, we don't cancel update a trustline we cancel a trustline update.

Reply:
okay, leave it, but we change the other updateTrustline then to requestTrustlineUpdate